core/src/main/java/nl/nn/adapterframework/filesystem/ExchangeFileSystem.java

```diff
@@ -56,19 +43,8 @@
 import microsoft.exchange.webservices.data.credential.ExchangeCredentials;
 import microsoft.exchange.webservices.data.credential.WebCredentials;
 import microsoft.exchange.webservices.data.credential.WebProxyCredentials;
-import microsoft.exchange.webservices.data.property.complex.Attachment;
-import microsoft.exchange.webservices.data.property.complex.AttachmentCollection;
-import microsoft.exchange.webservices.data.property.complex.EmailAddress;
-import microsoft.exchange.webservices.data.property.complex.EmailAddressCollection;
-import microsoft.exchange.webservices.data.property.complex.FileAttachment;
-import microsoft.exchange.webservices.data.property.complex.FolderId;
-import microsoft.exchange.webservices.data.property.complex.InternetMessageHeader;
-import microsoft.exchange.webservices.data.property.complex.InternetMessageHeaderCollection;
-import microsoft.exchange.webservices.data.property.complex.ItemAttachment;
-import microsoft.exchange.webservices.data.property.complex.ItemId;
-import microsoft.exchange.webservices.data.property.complex.Mailbox;
-import microsoft.exchange.webservices.data.property.complex.MessageBody;
-import microsoft.exchange.webservices.data.property.complex.MimeContent;
+import microsoft.exchange.webservices.data.misc.ImpersonatedUserId;
+import microsoft.exchange.webservices.data.property.complex.*;
```

Comment:
Druk een keer op Ctrl-Shift-O om de imports netjes te krijgen.

Reply:
Helaas werkt dat in IntelliJ toch net ietsje anders... Ik heb hiervoor ook iets van een shortcut gebruikt maar op een of andere manier was IntelliJ van mening dat het beter was om * te importeren...

Ik zal eens opzoek gaan of dit configurabel is want het is zonde van de tijd om dit soort foutjes te moeten corrigeren (en reviewen uiteraard)